src/Behat/Config/Config.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Behat\Config;
+
+final class Config
+{
+    public function __construct(
+        private array $settings = []
+    ) {
+    }
+
+    public function import(string ...$files): self
```

Comment:
I think this should be better without variadic thing or I should remame the method "imports" instead.
```suggestion
    public function import(string $file): self
```